File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/xrts/XRT1002U00GrdPaStatusInfo.java
package nta.med.data.model.ihis.xrts;

public class XRT1002U00GrdPaStatusInfo {
	private String bunho;
	private String patStatus;
	private String patStatusName;
	private String patStatusCode;
	private String patStatusCodeName;
	private String ment;
	private Double seq;
	private String indispensableYn;
	private String contKey;
	public XRT1002U00GrdPaStatusInfo(String bunho, String patStatus,
			String patStatusName, String patStatusCode,
			String patStatusCodeName, String ment, Double seq,
			String indispensableYn, String contKey) {
		super();
		this.bunho = bunho;
		this.patStatus = patStatus;
		this.patStatusName = patStatusName;
		this.patStatusCode = patStatusCode;
		this.patStatusCodeName = patStatusCodeName;
		this.ment = ment;
		this.seq = seq;
		this.indispensableYn = indispensableYn;
		this.contKey = contKey;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getPatStatus() {
		return patStatus;
	}
	public void setPatStatus(String patStatus) {
		this.patStatus = patStatus;
	}
	public String getPatStatusName() {
		return patStatusName;
	}
	public void setPatStatusName(String patStatusName) {
		this.patStatusName = patStatusName;
	}
	public String getPatStatusCode() {
		return patStatusCode;
	}
	public void setPatStatusCode(String patStatusCode) {
		this.patStatusCode = patStatusCode;
	}
	public String getPatStatusCodeName() {
		return patStatusCodeName;
	}
	public void setPatStatusCodeName(String patStatusCodeName) {
		this.patStatusCodeName = patStatusCodeName;
	}
	public String getMent() {
		return ment;
	}
	public void setMent(String ment) {
		this.ment = ment;
	}
	public Double getSeq() {
		return seq;
	}
	public void setSeq(Double seq) {
		this.seq = seq;
	}
	public String getIndispensableYn() {
		return indispensableYn;
	}
	public void setIndispensableYn(String indispensableYn) {
		this.indispensableYn = indispensableYn;
	}
	public String getContKey() {
		return contKey;
	}
	public void setContKey(String contKey) {
		this.contKey = contKey;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Xrts/XRT1002U00GrdPaStatusInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Xrts
{
    [Serializable]
    public class XRT1002U00GrdPaStatusInfo
    {
        private String _bunho;
        private String _patStatus;
        private String _patStatusName;
        private String _patStatusCode;
        private String _patStatusCodeName;
        private String _ment;
        private String _seq;
        private String _indispensableYn;
        private String _contKey;

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String PatStatus
        {
            get { return this._patStatus; }
            set { this._patStatus = value; }
        }

        public String PatStatusName
        {
            get { return this._patStatusName; }
            set { this._patStatusName = value; }
        }

        public String PatStatusCode
        {
            get { return this._patStatusCode; }
            set { this._patStatusCode = value; }
        }

        public String PatStatusCodeName
        {
            get { return this._patStatusCodeName; }
            set { this._patStatusCodeName = value; }
        }

        public String Ment
        {
            get { return this._ment; }
            set { this._ment = value; }
        }

        public String Seq
        {
            get { return this._seq; }
            set { this._seq = value; }
        }

        public String IndispensableYn
        {
            get { return this._indispensableYn; }
            set { this._indispensableYn = value; }
        }

        public String ContKey
        {
            get { return this._contKey; }
            set { this._contKey = value; }
        }

        public XRT1002U00GrdPaStatusInfo() { }

        public XRT1002U00GrdPaStatusInfo(String bunho, String patStatus, String patStatusName, String patStatusCode, String patStatusCodeName, String ment, String seq, String indispensableYn, String contKey)
        {
            this._bunho = bunho;
            this._patStatus = patStatus;
            this._patStatusName = patStatusName;
            this._patStatusCode = patStatusCode;
            this._patStatusCodeName = patStatusCodeName;
            this._ment = ment;
            this._seq = seq;
            this._indispensableYn = indispensableYn;
            this._contKey = contKey;
        }

    }
}